app/workers/process_domain_events_worker.rb

```diff
@@ -0,0 +1,49 @@
+class ProcessDomainEventsWorker
+  include Sidekiq::Worker
+
+  class_attribute :publisher
+  self.publisher = ->(*args) { Rails.application.config.event_store.publish_event(*args) }
```

Comment:
We are starting to to have this in many places and it's already refined in https://github.com/3scale/porta/blob/af47f5f0f34b856571eb0c3a4227ba5f17ed7849/app/events/base_event_store_event.rb#L3. Perhaps we could use it from there

Reply:
Yes, but this is not the time I think.

Reply:
I agree that it is not the time 😄 Btw I am doing a PR for refactoring these things in https://github.com/3scale/porta/pull/1023, so I can do it there 👍 but let's wait for after CR1 😄

Reply:
How is that so hard? Define in `BaseEventStoreEvent`

```ruby
def publish
  PUBLISHER.call(self)
end
```

Delete 2 lines of code here (4 and 5) and then

https://github.com/3scale/porta/blob/c62569f715823e343fc729c8a577e1958497b2b7/app/workers/process_domain_events_worker.rb#L11
becomes
```ruby
events.each(&:publish)
```

Reply:
No need to change it here if we are in such a hurry with the PR but it's a quite simpler change than what we expect to achieve with https://github.com/3scale/porta/pull/1023 and also about not adding more TODOs that we never address.